{"uuid": "e2d55e78-273a-4e3c-a2d7-8ed3febde22b", "children": ["e41feb29-2270-47ae-8ef1-f9fd8cea0088"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "RuntimeError: LOCAL_APK_PATH not found: C:\\projects\\mobile-demo\\apps\\my-demo-app-android.apk\nSet env var LOCAL_APK_PATH or place apk there.\n", "trace": "  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\projects\\mobile-demo\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\projects\\mobile-demo\\conftest.py\", line 78, in driver\n    driver = _create_local_android_driver()\n  File \"C:\\projects\\mobile-demo\\conftest.py\", line 17, in _create_local_android_driver\n    raise RuntimeError(f\"LOCAL_APK_PATH not found: {LOCAL_APK}\\nSet env var LOCAL_APK_PATH or place apk there.\")\n"}, "start": 1757864380892, "stop": 1757864380893}], "start": 1757864380892, "stop": 1757864381027}